### PR TITLE
#18 add grafana 7 compatibility

### PR DESCRIPTION
**Pull request by j-or**
#fix https://github.com/alexandrainst/alexandra-trackmap-panel/issues/18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,173 +1,117 @@
...
-    "type": "table"
-  }
-]
+# Track Map - Grafana Panel Plugin
+A map plugin to visualise coordinates as markers, hexbin, ant path, or heatmap.
+
+## Earlier versions
+This is a new version of the Track Map plugin - now built with React, which works with Grafana 7+.
+
+Note that this plugin is [not backwards compatible](https://grafana.com/docs/grafana/latest/developers/plugins/migration-guide/#compatibility-between-grafana-versions) with the old Grafana 6 versions.
```
Comment (Alkarex) [inline README.md:7]:
```suggestion
Note that this plugin is [not backwards compatible](https://grafana.com/docs/grafana/latest/developers/plugins/migration-guide/#compatibility-between-grafana-versions) with the old Grafana 6 versions.

For Grafana 6 and older, please use our [1.x branch](https://github.com/alexandrainst/alexandra-trackmap-panel/releases/tag/1.2.4) instead.
```

**Comment by nlubello** [comment]
Tested on Grafana 7.2.1 with influxDB query 
`SELECT "lat"::float / 10000000  AS "lat", "long"::float / 10000000  AS "lon" FROM "gps" WHERE ("device" = 'Telemetry_ESP32_0001') AND $timeFilter`

**Comment by usommerl** [comment]
Hi,

Thanks for working on Grafana 7 compatibility. I'm really looking forward to this!
I have tested [this PR](https://github.com/alexandrainst/alexandra-trackmap-panel/archive/%2318_add_grafana_7_compatibility.zip) with Grafana v7.2.1 and an InfluxDB data source successfully.

I have noticed that some features of v1.4.x are no longer present in this version. These features are not that important for my use case right now, but I will mention them here nonetheless:
- I'm no longer able to set the initial map center latitude/longitude and zoom level to the current value of the map via a button
- The configuration option "Re-center map" is missing

**Comment by Alkarex** [comment]
Thanks for the feedback @usommerl 👍 
I think that is good enough for a first release of this new 2.x branch.

**Comment by jsBlackBelt** [comment]
Hi, 

Thanks for working on this compatibility 7.x.x issue!
However, I don't see it in the grafana plugin store: https://grafana.com/grafana/plugins/alexandra-trackmap-panel
the newest version there is 1.2.5

When will 2.0.0 be released?

**Comment by usommerl** [comment]
@jsBlackBelt Please subscribe to this [PR](https://github.com/grafana/grafana-plugin-repository/pull/766).

**Comment by Alcarin** [comment]
Seems Grafana developer are waiting your feedback @Alkarex  ?

**Pull request merged**